clients/client-python/build.gradle.kts

```diff
@@ -38,14 +38,19 @@ tasks {
     args = listOf("install", "-e", ".[dev]")
   }
 
+  val pylint by registering(VenvTask::class) {
+    venvExec = "pylint"
+    args = listOf("./gravitino", "./tests")
+  }
```

Comment:
How do we trigger the pylint task for gradle, is it `./gradlew pylint`?

Reply:
I think this should not be resolved unless we have a consensus on this. 

I would suggest that we can trigger the pylint task implicitly before compile or other task, like spotless which will be triggered automatically when running `compile`, `build` or others.

Here you bring in a new task/command without documentation, it is not a good practice.

Reply:
Actually it will run before running integration tests of python. The `test` task depends on the `pylint` task. I will add pylint task before build task if necessary.

Reply:
@noidname01 sorry for late response because of public holiday. I would suggest we align with spotless, can be triggered before `compile`?